kore/src/Kore/Internal/TermLike.hs

```diff
@@ -1721,11 +1757,11 @@ pattern Builtin_
     -> TermLike variable
 
 pattern BuiltinBool_
-    :: Domain.InternalBool
+    :: InternalBool
     -> TermLike variable
 
-pattern BuiltinInt_
-    :: Domain.InternalInt
+pattern InternalInt_
```

Comment:
Why was only `BuiltinInt` renamed?

Reply:
Thanks for pointing this out! I simply forgot the others.